.github/workflows/publish.yaml

```
@@ -0,0 +1,38 @@
+name: Build and publish image
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+env:
+  IMAGE_PREFIX: 'docker.pkg.github.com/fairdatasociety/fdp-contracts'
+  BEE_ENV_PREFIX: 'swarm-test'
+  BLOCKCHAIN_VERSION: '1.2.0'
+  ENV_FILE: '.env'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Github authentication
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login https://docker.pkg.github.com -u ${GITHUB_ACTOR} --password-stdin
```

if everything works with the publishing, please change it to the following:
```suggestion
      - name: Docker authentication
        run: |
          echo "${{ secrets.DOCKER_TOKEN }}" | docker login -u ${{ secrets.DOCKER_USER }} --password-stdin
```